VisualStudio.gitignore

```diff
@@ -205,6 +205,9 @@ orleans.codegen.cs
 # (https://github.com/github/gitignore/pull/1529#issuecomment-104372622)
 #bower_components/
 
+# VS15: by default bower location is moved, bower restore is true
+**/wwwroot/lib/
```

Comment:
The default project setup in ASP.NET Core is that the bower directory is set to wwwroot/lib.  These are external resources that are restored regularly, so there should be no need to add these to VCS.  

The default intended use for other directories in wwwroot is to hold user code and static resources, which should be in VCS.